models/hooks.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /*
- * A bare-bones implementation of an ordered hook system, kinda like WordPress's.
+ * A bare-bones implementation of an ordered hook system, kind of like WordPress's.
```

Comment:
Replace "kind of like WordPress's"
to " similar to that of WordPress"

Reply:
I fixed line 3.